README.md

```diff
@@ -1,30 +1,51 @@
-<h1 align="center">A official AsyncAPI React Component</h1>
+# AsyncAPI React Component
 
 ## Overview
 
-A official [React](https://reactjs.org/) component for AsyncAPI specification. It allows to validation and render AsyncAPI specification and use own styles.
+This project is an official [React](https://reactjs.org/) component for AsyncAPI specification. It allows you to validate and render AsyncAPI specification using your own styles.
 
-## Philosophy
+### Project structure
 
+This repository has the following structure:
 
+```
+  ├── docs                        # Directory with project-related documents
+  ├── library                     # Structure of the AsyncApi React component
+  │    ├── src                    # Source code of the AsyncApi React component
+  │    │    ├── components        # Source code of generic components used in the "containers" directory
+  │    │    ├── config            # Configuration of the AsyncApi React component
+  │    │    ├── containers        # Subcomponents for specific parts of the AsyncApi React component
+  │    │    ├── helpers           # Various helper functions
+  │    │    └── theme             # Theme-related files for the AsyncApi React component
+  │    └── test                   # Tests for the AsyncApi React component
+  └── playground                  # Structure of the Playground application for the AsyncApi React component
+       ├── public                 # Fonts, images, and icons used in the Playground application
+       ├── src                    # Source code of the Playground application
+       │    ├── common            # Various helper functions including mocks
+       │    └── components        # Components used in the Playground application
+       └── test                   # Tests for the Playground application
+```
+
+## Prerequisites
+
+Use the following tools to develop the AsyncApi React component:
 
-## Live Demo
+* React (version 16.0.0 or higher)
+* [Styled-components](https://github.com/styled-components/styled-components) (version 4.0.0 or higher)
+* TypeScript (version 3.0.0 or higher)
 
-A [live playground](https://magicmatatjahu.github.io/asyncapi-react/) is hosted on gh-pages.
+## Installation
```

Comment:
please remove it
for me it is more a part of usage section really, 
so in usage, first step of usage is `npm install --save asyncapi-react` and second is https://github.com/kyma-incubator/asyncapi-react/pull/9/files#diff-04c6e90faac2675aa89e2176d2eec7d8R49